app/src/main/java/com/keylesspalace/tusky/TuskyApplication.java

```diff
@@ -39,6 +39,7 @@ public void onCreate() {
         Picasso.Builder builder = new Picasso.Builder(this);
         builder.downloader(new OkHttp3Downloader(OkHttpUtils.getCompatibleClient()));
         if (BuildConfig.DEBUG) {
+            builder.loggingEnabled(true);
```

Comment:
https://square.github.io/picasso/2.x/picasso/com/squareup/picasso/Picasso.Builder.html#loggingEnabled-boolean-

"WARNING: Enabling this will result in excessive object allocation. This should be only be used for debugging purposes. Do NOT pass BuildConfig.DEBUG."

Reply:
Ah, you already removed it, sorry